riotfile.py

```diff
@@ -670,13 +670,13 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pkgs={"pymongo": ["~=3.4", "~=3.11", "~=3.13"]},
                 ),
                 Venv(
-                    pys=select_pys(min_version="3.7", max_version="3.9"), pkgs={"pymongo": ["~=3.11", "~=4.0", latest]}
+                    pys=select_pys(min_version="3.7", max_version="3.9"), pkgs={"pymongo": ["~=3.11", "~=4.5", latest]}
                 ),
                 Venv(
                     # pymongo added support for Python 3.10 in 3.12.1
                     # pymongo added support for Python 3.11 in 3.12.3
                     pys=select_pys(min_version="3.10"),
-                    pkgs={"pymongo": ["~=3.12.3", "~=4.0", latest]},
+                    pkgs={"pymongo": ["~=3.12.3", "~=4.5", latest]},
```

Comment:
Why was this bump necessary? Did the patching code work fine with `pymongo>=4.0,<4.5`? Shouldn't `~=4.0` resolve to the same version as `~=4.5`?

Reply:
Yeah, it does. Not sure why it wasn't working yesterday.